src/pages/basic_data/institution_admin/person_register/PersonRegisterPage.tsx

```diff
@@ -47,7 +47,7 @@ export const PersonRegisterPage = () => {
     queryKey: ['employees', user?.topOrgCristinId, rowsPerPage, page, debouncedSearchQuery, searchQuery],
     queryFn: ({ signal }) =>
       user?.topOrgCristinId
-        ? fetchEmployees(user?.topOrgCristinId, rowsPerPage, page, debouncedSearchQuery, signal)
+        ? fetchEmployees(user?.topOrgCristinId, rowsPerPage, page, 'k' /*debouncedSearchQuery*/, signal)
```

Comment:
?

Reply:
dette er bare en draft PR, som jeg bruker å lage når jeg starter en ny branch, bare for å holde følge på hvor mange filer jeg har endret osv, for å holde litt oversikt om PRen begynner å bli litt for stor. Alle draft PRer med "WIP" (work in progress) i description kan egentlig ses bort i fra. Gjenstår flere dagers arbeid før dette er noe jeg er komfortabel med å merge, så vil bli store endinger.

Akkurat `'k'` der er bare for at jeg skal få treff på meg selv i tillegg til andre hver gang, uten at jeg manuelt må skrive i søkefeltet. Siden vi kjører squash merge vil ikke dette vises i git-historikken etter merge :)